content/en/hosting/4.x/production/docker/_index.md

```diff
@@ -1,17 +1,160 @@
+
+To follow the progress tail the log of the upgrade service container by running this:
+
+`docker logs -f upgrade-service_cht-upgrade-service_1`
+
+To make sure everything is running correctly, call `docker ps` and make sure that 7 CHT containers show:
+
+```shell
+CONTAINER ID   IMAGE                                                         COMMAND                   CREATED          STATUS         PORTS                                                                      NAMES
+8c1c22d526f3   public.ecr.aws/s5s3h4s7/cht-nginx:4.0.1-4.0.1                 "/docker-entrypoint.…"    17 minutes ago   Up 8 minutes   0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:443->443/tcp, :::443->443/tcp   cht_nginx_1
+f7b596be2721   public.ecr.aws/s5s3h4s7/cht-api:4.0.1-4.0.1                   "/bin/bash /api/dock…"    17 minutes ago   Up 8 minutes   5988/tcp                                                                   cht_api_1
+029cd86ac721   public.ecr.aws/s5s3h4s7/cht-sentinel:4.0.1-4.0.1              "/bin/bash /sentinel…"    17 minutes ago   Up 8 minutes                                                                              cht_sentinel_1
+61ee1e0b377b   public.ecr.aws/s5s3h4s7/cht-haproxy-healthcheck:4.0.1-4.0.1   "/bin/sh -c \"/app/ch…"   17 minutes ago   Up 8 minutes                                                                              cht_healthcheck_1
+87415a2d91ea   public.ecr.aws/s5s3h4s7/cht-haproxy:4.0.1-4.0.1               "/entrypoint.sh -f /…"    17 minutes ago   Up 8 minutes   5984/tcp                                                                   cht_haproxy_1
+58454457467a   public.ecr.aws/s5s3h4s7/cht-couchdb:4.0.1-4.0.1               "tini -- /docker-ent…"    17 minutes ago   Up 8 minutes   4369/tcp, 5984/tcp, 9100/tcp                                               cht_couchdb_1
+d01343658f3f   public.ecr.aws/s5s3h4s7/cht-upgrade-service:latest            "node /app/src/index…"    17 minutes ago   Up 8 minutes                                                                              upgrade-service-cht-upgrade-service-1
+```
+
+This should show related to the CHT core are running
+* cht_nginx
+* cht_api
+* cht_sentinel
+* cht_couchdb
+* cht_healthcheck
+* cht_haproxy
+* cht-upgrade-service
+
+Take note of the `STATUS` column and make sure no errors are displayed there. If any container is restarting or mentioning any other error, check the logs using the `docker logs <container-name>` command.
+
+If all has gone well, nginx should now be listening at both port 80 and port 443. Port 80 has a permanent redirect to port 443, so you can only access the CHT using https.
+
+To login as the `medic` user in the web app, you can find your password with this command:
+   ```shell
+   grep COUCHDB_PASSWORD /home/ubuntu/cht/upgrade-service/.env | cut -d'=' -f2
+   ```
```

Comment:
The "suggestion" syntax is failing because of the trailing backticks but remove the whitespace at the start of these 3 lines IMO.

Reply:
fixed in b4f3aecd


-------

NB - this is how to do nested backticks for suggestions - just add more sets of 3!  So here I have 9 outer, 6 inner, and 3 innermost to show you what to do:

`````````
``````suggestion
```shell
grep COUCHDB_PASSWORD /home/ubuntu/cht/upgrade-service/.env | cut -d'=' -f2
```
``````
`````````

Where as just 6 outer and 3 inner yields the suggestion you were after:

``````suggestion
```shell
grep COUCHDB_PASSWORD /home/ubuntu/cht/upgrade-service/.env | cut -d'=' -f2
```
``````